src/pages/install/index.md

```diff
@@ -1,134 +1,19 @@
-2. Follow the "release page" link below "GitHub links", and grab either of the `win32` (for 32-bit Windows) or `win64` (for 64-bit Windows) `.zip` files, near the bottom of the page.
-3. Unzip that file, you should get the `.exe` files alongside a couple of `.dll`s.
-4. Either:
-   - ...put all of the files in a directory, then add it to the `PATH`.
-     This will **permanently** allow you to use RGBDS **from anywhere**.
-
-     <Tabs>
-     <TabItem value="gui" label="Graphically">
-
-       1. Open Control Panel
-       2. Click "User Accounts"
-       3. Click "User Accounts" again
-       4. Click "Change my environment variables"
-       5. Select the "Path" line in the **top** panel
-       6. Click "Edit..."
-       7. Click "Browse...", select the folder the files are in, and click OK
-       8. Make sure that the new entry (which should be highlighted) is at the bottom of the list; if not, click on "Move Down" until it is
-       9. Click "OK"
-       10. Click "OK"
-
-     </TabItem>
-     <TabItem value="cmd.exe" label="cmd.exe">
-
-       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
-
-       ```cmd
-       setx PATH "%PATH%<rgbds_path>;"
-       ```
-
-       ...then open a new window for the changes to take effect.
-
-       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
-
-     </TabItem>
-     <TabItem value="pwsh" label="PowerShell">
-
-       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
-
-       ```cmd
-       setx PATH ${Env:PATH}<rgbds_path>;
-       ```
-
-       ...then open a new window for the changes to take effect.
-
-       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
-
-     </TabItem>
-     </Tabs>
-
-   - ...or put all of the files in your project's directory.
-   - ...or put all of the files in a directory already in the `PATH`.
-5. Profit! RGBDS can now be used from your favorite command line (`cmd.exe` or PowerShell, most likely). You can test it by running `rgbasm --version`.
+Most people will want to use a stable release (the more recent the better).
 
-</TabItem>
-</Tabs>
+These are the operating systems for which pre-built executables are available:
+- [Linux](/install/linux)
+- [macOS](/install/macOS)
+- [Windows](/install/windows)
 
-## Other
```

Comment:
why remove the chapter "other" here, that was contianing e.g. docker as well?

Reply:
It seemed to introduce extra nesting that didn't serve much of a purpose. Considering how short each section is, and that this page is essentially a dispatcher, I think vertical space is precious.